### DISCUSSION TITLE
react-window (FixedSizeList and VariableSizeGrid )

### DISCUSSION THREAD

**Discussion by aaronpaulmallorca** [npm/feedback#725] [General]
Hi,

We used the FixedSizeList and VariableSizeGrid of the react-window and found an issue of broken styling when we load a large data (e.g.: 1million) 
I used the demo page and did some adjustment in the code to make it simple data. (https://codesandbox.io/s/github/bvaughn/react-window/tree/master/website/sandboxes/variable-size-grid?file=/index.js)

The issue here was the line in the row starting Row-349527, I wonder where that line come from because I didn't put any border or any style to produce that line. Is there anyone here encounter the similar issue? 

![image](https://user-images.githubusercontent.com/7484085/177950161-f83f8bbc-ff8d-4d93-9ffa-a2f4810fed15.png)

**Comment by ljharb** [comment]
You'll need to report this to the packages directly; this repo is for providing feedback to npm about its website and registry.
